lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -91,6 +91,7 @@ export default function SubmitGradeForm({ disabled = false, student }) {
   // Clear the previous grade saved status when the user changes.
   useEffect(() => {
     setGradeSaved(false);
+    inputRef.current.focus();
```

Comment:
Will this (try to) focus the input field when the _no_ `student` is selected? That is, if you have a student selected, and then page back to "All Students", will this component attempt to focus the input field?

Reply:
It will, but won't matter since its disabled.  Kind of a no-op, but a nice one because we don't need a if condition.

Reply:
Thanks for the feedback, select makes sense and it was not too bad with `useLayoutEffect` getting trigged on a change to the grade state -- everything fell into place.